hs_collection_resource/management/commands/fix_collection_version_missing_ispartof.py

```diff
@@ -32,4 +34,35 @@ def handle(self, *args, **options):
                         res.metadata.create_element('relation',
```

Comment:
Should we be checking that the IsPartOf relation does not exist before creating a new one?

Reply:
yep thanks for looking!